src/databricks/labs/lsql/backends.py

```diff
@@ -372,6 +372,18 @@ def rows_written_for(self, full_name: str, mode: str) -> list[DataclassInstance]
             rows += stub_rows
         return rows
 
+    def has_rows_written_for(self, full_name: str) -> bool:
```

Comment:
Although I think I came up with this name, I'm not sure if there's something better? What troubles me is that the name is misleading for when `[]` was written: the plain-language description of this situation is that the table has been written but doesn't contain any rows for the table, but this doesn't really match the function name.

Any ideas?